RELEASE_CHECKLIST.md

```diff
@@ -25,30 +25,31 @@ in lieu of an official changelog.
     breaking change, also add a BREAKING label.
 4. Set the release date in `NEXT_CHANGELOG.md`. Add this release to the
     `CHANGELOG.md`. Use the structure of previous entries.
-5. Update `docker.mdx` and `kubernetes.mdx` with the release version.
-6. Update `helm/chart/router/Chart.yaml` and in `helm/chart/router/README.md` as follows:
-   - increment the version. e.g. `version: 0.1.2` becomes `version: 0.1.3`
-   - update the appVersion to the release version. e.g.: `appVersion: "v0.9.0"`
-7. cd helm/chart && helm-docs router; cd - (if required, install [helm-docs](https://github.com/norwoodj/helm-docs))
-8. Update `federation-version-support.mdx` with the latest version info. Use https://github.com/apollographql/version_matrix to generate the version matrix.
-9. Update the version in docker-compose files in `dockerfiles` directory.
-10. Update the license list with `cargo about generate --workspace -o licenses.html about.hbs`.
-    You can install `cargo-about` by running `cargo install cargo-about`.
 
 ### Start a release PR
 
 1. Make sure you have `cargo` installed on your machine and in your `PATH`.
 2. Create a new branch "#.#.#" where "#.#.#" is this release's version
     (release) or "#.#.#-rc.#" (release candidate)
-3. Update the version in `*/Cargo.toml` (do not forget the ones in scaffold templates).
-4. Add a new section in `CHANGELOG.md` with the contents of `NEXT_CHANGELOG.md`
-5. Put a Release date and the version number on the new `CHANGELOG.md` section
-4. Update the version in `NEXT_CHANGELOG.md`.
-5. Clear `NEXT_CHANGELOG.md` leaving only the template.
-6. Run `cargo check` so the lock file gets updated.
-7. Run `cargo xtask check-compliance`.
-8. Push up a commit with all the changes. The commit message should be "release: v#.#.#" or "release: v#.#.#-rc.#"
-9. Request review from the Router team.
+3. Update the `version` in `*/Cargo.toml` (do not forget the ones in scaffold templates).
+   - Be certain to also update the ones in the `scaffold` templates
+4. Update `docker.mdx` and `kubernetes.mdx` with the release version.
+5. Update `helm/chart/router/Chart.yaml` and in `helm/chart/router/README.md` as follows:
+   - increment the version. e.g. `version: 0.1.2` becomes `version: 0.1.3`
+   - update the appVersion to the release version. e.g.: `appVersion: "v0.9.0"`
+6. cd helm/chart && helm-docs router; cd - (if required, install [helm-docs](https://github.com/norwoodj/helm-docs))
```

Comment:
Maybe we should expand step 1 with all the things you need to have installed?

I think the list is:
 - cargo
 - cargo-about (plugin)
 - helm-docs
 - rustc (via rustup or whatever)
 - ...

We don't mention that you need rust right now, but you won't be able to run xtask without it...

Reply:
I like the suggestion, but wouldn't that be a `DEVELOPMENT.md` concern?  Those seem like commands you need to do development not just releasing (and I would expect development should be a requirement/precursor to releasing)

Reply:
Also, I now realize you're talking about an actual step 0, not using `0.` to get around explicit numbering.  (For context: We discussed this on an audio call.)